heatmap_gif.py

```diff
@@ -0,0 +1,87 @@
+import requests
+import argparse
+import sys
+import os
+import json
+
+import seaborn
+import numpy
+import matplotlib.pyplot
+
+import imageio
+class HeatmapGif:
+#Set up workspace for API calls to Rhombus Systems
+    def __init__(self,args):
+        #Initialize argument parser
+        self.argParser = self.__initArgParser()
+        self.args = self.argParser.parse_args(args)
+        #Create a session to set default call Header information
+        self.session = requests.session()
+        self.session.headers = {"x-auth-scheme": "api-token","x-auth-apikey": self.args.apiKey}
+        #Set a default base URL for api calls
+        self.url = "https://api2.rhombussystems.com/api/"
+
+    @staticmethod
+    def __initArgParser():
+        argParser = argparse.ArgumentParser(description="Produces a heatmap GIF summarizing activity across the seelcted timespan")
+        argParser.add_argument("apiKey", help="API Access Key, get this form your console.")
+        argParser.add_argument("uuid", help = "UUID identifies the target camera")
+        argParser.add_argument("startTime", help = "Time in seconds since epoch to start sampling")
+        argParser.add_argument("endTime", help = "Time in seconds since epoch to stop sampling")
+        argParser.add_argument("stepSize", help = "Time in seconds represented by each frame")
+        return argParser
+
+    def execute(self):
+        stepSize = int(self.args.stepSize)
+        prevTime = int(self.args.startTime)
+        endTime = int(self.args.endTime)
+        curTime = stepSize+prevTime
+        
+        ##Setup writer to output heatmap.gif
+        with imageio.get_writer('./heatmap.gif',mode='I') as writer:
+            while(curTime <= endTime):
```

Comment:
A nice quality of life update here would be to add a progress bar or just print the percentage complete.  Since this could run for a while, it's nice to have feedback that it is in fact progressing